runtime/src/iouring/mod.rs

```diff
@@ -115,25 +115,12 @@ pub struct Config {
     /// See IORING_SETUP_SINGLE_ISSUER in <https://man7.org/linux/man-pages/man2/io_uring_setup.2.html>.
     pub single_issuer: bool,
     /// In the io_uring event loop (`run`), wait at most this long for a new
-    /// completion before checking for new work to submit to the io_ring.
```

Comment:
I removed the deadlock explanation because felt it was unnecessary if it can't happen anymore.